package.json

```diff
@@ -25,6 +25,7 @@
     "test-coverage": "nyc --per-file mocha --compilers js:babel-core/register --require source-map-support/register --full-trace --bail src/**/*.test.js"
   },
   "dependencies": {
+    "unixify" :"1.0.0"
```

Comment:
Had to take a dependency on unixify. it's not large and could be inlined, but it was critical to me. (and memfs or unionfs needs it anyway)